index.js

```diff
@@ -118,8 +118,10 @@ const timeout = (promise, ms, abortController) => new Promise((resolve, reject)
 	/* eslint-disable promise/prefer-await-to-then */
 	promise.then(resolve).catch(reject);
 	delay(ms).then(() => {
+		if (abortController) {
```

Comment:
```suggestion
		if (supportsAbortController) {
```

Reply:
timeout() is called with `this.abortController`, which is undefined if `supportsAbortController` is falsey. To avoid bugs down the line I think it's better to test for `abortController` here, not `supportsAbortController` because the method is being called on the instance. But it doesn't really make any difference to the functionality of the code as-is, just a style preference I guess.

Reply:
Also did it like this because this was how the test was done before #122

Reply:
Fixed. 👍 

Unrelated: For some reason GitHub's "Commit suggestion" feature said I don't have write access, even though I do (including to the fork). Strange.

Reply:
> I would be fine with removing the supportsAbortController variable altogether in favor of if (abortController) and if (AbortController). To me, the latter is slightly simpler and doesn't really have any serious downsides.

👍